organization/financial-model.md

```diff
@@ -2,21 +2,21 @@
 type: Resource
 ---
 
-# Financing model for the Foundation for Public Code
+# Financial model for the Foundation for Public Code
 
-This conceptual model helps founding members shape the Foundation's financial model.
+This content outlines the objectives of the Foundation's financial model.
 
 ## Overview
 
 The Foundation for Public Code is a member-owned association. Its mission is to enable public-purpose software and policy that is open and collaborative.
 
-This means we are primarily member-funded, with additional support from grants, subsidies and philanthropic donations.
+We are currently wholly financially supported by philanthropists who share our commitment to our mission. To prevent undue influence, all donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch public benefit organization (ANBI). [See our philanthropy policy](../../organization/philanthropy.md) for more information about how we work with philanthropists and donations.
```

Comment:
```suggestion
We are currently wholly financially supported by philanthropists who share our commitment to our mission. To prevent undue influence, all donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization (ANBI)](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/what_is_pbo/). [See our philanthropy policy](../../organization/philanthropy.md) for more information about how we work with philanthropists and donations.
```